cub/benchmarks/bench/copy/memcpy.cu

```diff
@@ -184,7 +184,7 @@ void copy(nvbench::state& state,
   using buffer_offset_t    = std::uint32_t;
   using block_offset_t     = std::uint32_t;
 
-  constexpr bool is_memcpy = true;
+  constexpr cub::CopyAlg is_memcpy = cub::CopyAlg::Memcpy;
```

Comment:
Suggestion: inline this constant into the instantiation of `DispatchBatchMemcpy` below.